src/vite-types.d.ts

```diff
@@ -11,7 +11,7 @@ interface DbContribution {
 interface DbRepo {
   readonly id: number;
   readonly user_id?: number;
-  readonly stars: number;
```

Comment:
this should stay a number. is anything breaking without it?

Reply:
nothing broke but the data brought from the api was in string form not number form

Reply:
all the data coming from the backend is json so it would look like strings, the entities should be even with the backend models, in this case the contribution entity: https://github.com/open-sauced/api.opensauced.pizza/blob/beta/src/contribution/contribution.entity.ts

Tried doing: 
```
curl -X 'GET' \
  'https://api.opensauced.pizza/v1/repos/28457823/contributions?page=1&limit=10' \
  -H 'accept: application/json'
```

On second look, backend for contributions is not returning any kind of stars data so it should actually be removed entirely 😅

Reply:
hmm? it does return stars for me
![image](https://user-images.githubusercontent.com/52013393/205307976-7f4a4c54-ccbf-41af-a3ac-a447a694c81a.png)